app/models/user.rb

```diff
@@ -12,14 +12,11 @@ class User < ActiveRecord::Base
   validates :last_name, presence: true
   validates :username, presence: true
 
-  enum role: %w(default admin)
-
-  def admin?
-    role == "admin"
-  end
+  enum role: %w(default admin super_admin)
 
   def self.find_or_create_by_auth(auth_data)
-    user = User.where(provider: auth_data['provider'], id: auth_data['uid']).first_or_create
+    user = User.where(provider: auth_data["provider"], id: auth_data['uid'])
+               .first_or_create
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.